apps/meteor/app/lib/server/methods/getChannelHistory.ts

```diff
@@ -67,7 +67,9 @@ Meteor.methods<ServerMethods>({
 			throw new Meteor.Error('error-invalid-date', 'Invalid date', { method: 'getChannelHistory' });
 		}
 
-		const hiddenMessageTypes = getHiddenSystemMessages(room);
+		const hiddenSystemMessages = (await Settings.getValueById('Hide_System_Messages')) as MessageTypesValues[];
```

Comment:
Use `settings.get`

I'm referring to the cached settings, not the settings model